src/Common/Integrations/Custom_Table_Abstract.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Abstract for Custom Tables.
+ *
+ * @since TDB
+ *
+ * @package TEC\Common\Integrations
+ */
+
+namespace TEC\Common\Integrations;
```

Comment:
Why integrations?

Reply:
Not sure if Integrations was the best choice... This is just coming from master. Its released already.